IdentityCore/src/MSIDError.h

```diff
@@ -139,6 +139,7 @@ typedef NS_ENUM(NSInteger, MSIDErrorCode)
     MSIDErrorServerUnauthorizedClient   = -51414,
     MSIDErrorServerDeclinedScopes       = -51415,
     MSIDErrorServerAccessDenied         = -51416,
+    MSIDErrorServerError                = -51417,
```

Comment:
If you're introducing a new top level error here, MSAL needs to be updated accordingly as well, and mapping between MSID and MSAL

Reply:
Thank you for reminding me about this, I will update this when we update the I will update the submodule into the MSAL.